databusclient/api/delete.py

```
@@ -0,0 +1,187 @@
+
+    json_dict = json.loads(artifact_body)
+    versions = json_dict.get("databus:hasVersion")
+
+    # Single version case {}
+    if isinstance(versions, dict):
+        versions = [versions]
+    # Multiple versions case [{}, {}]
+
+    version_uris = [v["@id"] for v in versions if "@id" in v]
+    if not version_uris:
+        raise ValueError("No versions found in artifact JSON-LD")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)